spec/unit/prepare.spec.js

```diff
@@ -493,7 +494,7 @@ describe('prepare', () => {
         });
     });
 
-    describe('prepareIcons method', function () {
+    describe('prepareIcons', function () {
```

Comment:
Arrow functions shouldn't converted to `function ()` style.

Additionally this change appears to be unrelated to the problem this PR is trying to solve. PRs should be concise to only include changes required to address a problem.

Reply:
all non related changes have been reverted